CI/steam/scripts/install.bat

```diff
@@ -0,0 +1,85 @@
+@echo off
+@cd /d "%~dp0"
+
+goto checkAdmin
+
+
+:checkAdmin
+	net session >nul 2>&1
+	if %errorLevel% == 0 (
+		echo.
+	) else (
+		echo Administrative rights are required, please re-run this script as Administrator.
```

Comment:
"Administrative rights are required. Please re-run this script as Administrator."

Reply:
This is taken straight from https://github.com/obsproject/obs-studio/blob/master/plugins/win-dshow/virtualcam-install.bat.in#L10 so I'd suggest updating it there first.